business_logic/participant/participant_manipulated_data/participantSpecifiedTwitterData.js

```diff
@@ -1,52 +1,42 @@
 const twitterComm = require("../../twitter_communicator/twitterCommunicator")
-const manipulator = require("../manipulator/manipulator.js")
-const database = require("../../db/DBCommunicator.js");
+const twitterInnerApiUtils = require("../../twitter_communicator/twitter_internal_api/twitterInnerApiUtils")
 
 async function getUser(username){ // Later we will also send "praticipant" obj for manipulations.
```

Comment:
time to delete this comment?

Reply:
The one with "Later we will..."? Yea I'll delete it